assets/css/libs/_ribbon.sass

```diff
@@ -67,12 +67,12 @@ $ribbon-border: .1rem solid $grey-lighter !default
 
 .corner-top-right-ribbon
```

Comment:
@tajchumber rename this css class, now the component will be placed at the left side of the div, so does not make sense anymore the name be `corner-top-RIGHT-ribbon`

Reply:
of course! thanks picking this up!